mediawiki.js

```diff
@@ -35,4 +40,15 @@
 			}
 		}
 	]
-}
+};
+
+// Extend max-len.ignorePattern
+config.rules[ 'max-len' ] = commonRules.rules[ 'max-len' ];
+config.rules[ 'max-len' ][ 1 ].ignorePattern =
+	'(' +
+		config.rules[ 'max-len' ][ 1 ].ignorePattern +
+		'|' +
+		'[\\s]*(//|<!--) \\* [a-z-]+$' +
```

Comment:
I think we should also allow numbers and uppercase letters in message names. Why not use the same regex that msg-doc [uses internally](https://github.com/wikimedia/eslint-plugin-mediawiki/blob/master/src/utils.js#L13)? That one is much more permissive.

Reply:
Yes, we should allow numbers. IIRC upper case letters are not supposed to be used in message keys and can cause problems, although they have been used in some extensions erroneously so maybe we should allow them.

That pattern you link too is very loose because it allows things with spaces like "* label-type-0, label-type-1, etc.". For a line length rule we shouldn't be allowing spaces.